frontend/src/v4/routes/viewer3D/calibration3DInfoBox/calibration3DInfoBox.component.tsx

```diff
@@ -0,0 +1,48 @@
+/**
```

Comment:
Can we impose the standard padding to the buttons? 
![image](https://github.com/3drepo/3drepo.io/assets/11945337/d7c8eb91-e72b-496b-8c27-54b05dedb424)

Reply:
I don't know if this is relevant to this PR (if not please raise issue/move this comment as appropritate, or ignore if it's fixed elsewhere) , but the panel UIS should be on top of the viewer (ie. the dialog prompt and the bottom toolbar) UI:
![image](https://github.com/3drepo/3drepo.io/assets/11945337/12c4bad9-ff46-4986-8cac-afa00cd4a16c)

Reply:
Can we just hide the back button if it's disabled 
![image](https://github.com/3drepo/3drepo.io/assets/11945337/112d30f0-7e75-4daa-9631-a473e08dab8a)

Reply:
Can we stretch this out? 
![image](https://github.com/3drepo/3drepo.io/assets/11945337/26fa1fc0-9f19-425c-a8c0-5d510f056f62)

Reply:
This line is very out of place  since it uses the same colour but the top one has a gradient?
![image](https://github.com/3drepo/3drepo.io/assets/11945337/719542d6-3bd1-4b33-b05e-a06bfd8b2354)

Can we use secondary mid or secondary light, or it needs a gradient imposed like the top bar. (maybe check with troy)

Reply:
![image](https://github.com/3drepo/3drepo.io/assets/11945337/05a296b5-a09a-467b-947a-ec401d68759a)
Can we try the `label` typography here?

Reply:
![image](https://github.com/3drepo/3drepo.io/assets/11945337/b1176dc5-0b0d-4f57-8d78-f5a17170a30e)
This doesn't look very nice 😆 

Can we see what it looks like if you change the number to primyar main as well?

Reply:
![image](https://github.com/3drepo/3drepo.io/assets/11945337/62f37e23-22f4-4e1d-a060-3126e84fc263)
I feel like either the i icon needs to be centered vertically or the caption needs to be under it. Can you check with Troy what he thinks?

Reply:
> This line is very out of place since it uses the same colour but the top one has a gradient? ![image](https://private-user-images.githubusercontent.com/11945337/341099418-719542d6-3bd1-4b33-b05e-a06bfd8b2354.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTg5NjQxNzEsIm5iZiI6MTcxODk2Mzg3MSwicGF0aCI6Ii8xMTk0NTMzNy8zNDEwOTk0MTgtNzE5NTQyZDYtM2JkMS00YjMzLWIwNWUtYTA2YmZkOGIyMzU0LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MjElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjIxVDA5NTc1MVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWI0ZTFiMjJhYjNmMDliMDI1YWRkZGRlMmY0MTFiZDI4NDIyZjcyYjRhNmM4ZjE4MGFiMWJjMzU5ZDQwNzQzMjUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.ZdMdheoeiuW7t4WEmkkX8jfteQQxjfBkkEpeo5Ugo-U)
> 
> Can we use secondary mid or secondary light, or it needs a gradient imposed like the top bar. (maybe check with troy)

Happy for it to have the same gradient as the one above.

Reply:
> ![image](https://private-user-images.githubusercontent.com/11945337/341100739-05a296b5-a09a-467b-947a-ec401d68759a.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTg5NjQxNzEsIm5iZiI6MTcxODk2Mzg3MSwicGF0aCI6Ii8xMTk0NTMzNy8zNDExMDA3MzktMDVhMjk2YjUtYTA5YS00NjdiLTk0N2EtZWM0MDFkNjg3NTlhLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MjElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjIxVDA5NTc1MVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTk5ZTQwMTVkMWI2OGM5NGYzMGMwYmJlODAwZTExZWQ1MjdjODU4ZDA2Y2RiNjExODRlNWM1MzNmNDI3NTYzYjUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.knFxXOAg6MNqOpFp-srwv_p-Oqc2qWpE1gyarbMyDNg) Can we try the `label` typography here?

I have no problems with that.

Reply:
> ![image](https://private-user-images.githubusercontent.com/11945337/341101185-b1176dc5-0b0d-4f57-8d78-f5a17170a30e.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTg5NjQxNzEsIm5iZiI6MTcxODk2Mzg3MSwicGF0aCI6Ii8xMTk0NTMzNy8zNDExMDExODUtYjExNzZkYzUtMGIwZC00ZjU3LThkNzgtZjVhMTcxNzBhMzBlLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MjElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjIxVDA5NTc1MVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPWU2MDg0YjdmOTNiOTlkNzU0ZjZkMzc5NzczMzBiYzEyMjJmNWIwMThkMDQxYTk4YTExN2Q3MWZmM2QwYTEzNDUmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.iqrY0hKUZi2Mx33mlmu2izPbvQDdncxsOyjEf7AYw-Q) This doesn't look very nice 😆
> 
> Can we see what it looks like if you change the number to primyar main as well?

No in the primary the number almost vanishes. I would prefer to keep it that colour so its easier to read.

Reply:
> ![image](https://private-user-images.githubusercontent.com/11945337/341101853-62f37e23-22f4-4e1d-a060-3126e84fc263.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MTg5NjQxNzEsIm5iZiI6MTcxODk2Mzg3MSwicGF0aCI6Ii8xMTk0NTMzNy8zNDExMDE4NTMtNjJmMzdlMjMtMjJmNC00ZTFkLWEwNjAtMzEyNmU4NGZjMjYzLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA2MjElMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNjIxVDA5NTc1MVomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTQ4NGVlMTYxNDRjNDZmNGFiZjIyOGYwMGQ3YzBkZjAxYjZhYjJjMjI0ZTYwZTg1M2UxY2ZlNTczYzgyMmM1ZjgmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.E2UDXWyAS0qYltqELLxyzcmULEBtdIPqXMTOzuatykY) I feel like either the i icon needs to be centered vertically or the caption needs to be under it. Can you check with Troy what he thinks?

So this was based on style of components we currently have

![image](https://github.com/3drepo/3drepo.io/assets/1716684/558255ba-2a13-4ae7-b917-af66d008484c)

![image](https://github.com/3drepo/3drepo.io/assets/1716684/b1591fea-0692-4317-9696-07e82591856d)

I believe the icon should be at the top. My reason being that it is an info box and if it is for example an instruction it could have more text in it so the icon would look out of place when vertically centred.